ArjunaJTA/jta/classes/com/arjuna/ats/jta/common/JTAEnvironmentBean.java

```diff
@@ -1338,4 +1339,44 @@ public boolean isTransactionToThreadListenersEnabled() {
 	public void setTransactionToThreadListenersEnabled(boolean transactionToThreadListenersEnabled) {
 		this.transactionToThreadListenersEnabled = transactionToThreadListenersEnabled;
 	}
+
+    /**
+     * Returns the set of XAResourceRecovery implementation class names,
```

Comment:
Should not the comment talk about the `set of XAResource implementation class names` and not about `XAResourceRecovery`?

Reply:
You are correct. Try my latest push instead.